crates/cli_bin/tests/patterns_test.rs

```diff
@@ -309,3 +319,106 @@ fn tests_python_pattern_with_file_name() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn patterns_test_watch_mode_case_patterns_changed() -> Result<()> {
+    let (tx, rx) = mpsc::channel();
+
+    let (temp_dir, temp_grit_dir) = get_fixture(".grit", false)?;
+    let test_yaml_path = temp_grit_dir.join("grit.yaml");
+    let temp_dir_path = temp_dir.path().to_owned();
+
+    let _cmd_handle = thread::spawn(move || {
+        let mut cmd = Command::cargo_bin(BIN_NAME)
```

Comment:
Use `get_test_cmd()` instead and avoid hard-coding the BIN_name.

Reply:
I did try to use `get_test_cmd()`, but was not able to use `.stdout()` with it. 

was getting below error:
`no method named `stdout` found for mutable reference `&mut assert_cmd::Command` in the current scope
method not found in `&mut Command``

Reply:
created get_test_process_cmd() to use process::Command, which supports .stdout()

https://github.com/getgrit/gritql/pull/298/commits/58957570dba4313b4980c1d6e6053f3ef861c6dd

Reply:
We have many other tests that inspect stdout: https://github.com/getgrit/gritql/blob/f850d3aaaefeefedad2446a230728636632b1484/crates/cli_bin/tests/help.rs#L23

I don't understand why you need a new interface.